### PR TITLE
Admin namespace remove phase ii

### PR DESCRIPTION
**Pull request by tchorzewski1991**
Some small changes to friendlyID and finally remove admin routes.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -20,4 +21,12 @@ class User < ActiveRecord::Base
   # def name
   #   "#{fname} #{lname}" if fname.present? or lname.present?
   # end
+
+  def should_generate_new_friendly_id?
```
Comment (holden) [inline app/models/user.rb]: Why is this needed?

File: app/models/user.rb [hunk 2]
```diff
@@ -20,4 +21,12 @@ class User < ActiveRecord::Base
...
+    changed?
+  end
+
+  def slug_candidates
```
Comment (holden) [inline app/models/user.rb]: I think just username itself is fine, unless there's something i'm not aware of.  Shouldn't need a candidate method.

Comment (tchorzewski1991) [inline app/models/user.rb]: So username should be required?  I made this because username is optional, and what if there is no username? The same "should_generate... " method is needed for this

Comment (holden) [inline app/models/user.rb]: Yeah, just require it.

File: app/models/user.rb [hunk 3]
```diff
@@ -23,10 +25,6 @@ class User < ActiveRecord::Base
   # end
```
Comment (tchorzewski1991) [inline app/models/user.rb]: I suppose this method should stay, because it allow us to update slug when we update our username. This method is provided by FriendlyId gem.

File: app/models/user.rb [hunk 4]
```diff
@@ -23,10 +25,6 @@ class User < ActiveRecord::Base
...
 
   def should_generate_new_friendly_id?
-    changed?
-  end
-
-  def slug_candidates
-    [:username, :id]
+    username_changed?
   end
```
Comment (tchorzewski1991) [inline app/models/user.rb]: I suppose this method should stay, because it allow us to update slug when we update our username. This method is provided by FriendlyId gem.
